### DISCUSSION TITLE
BIOS 0056 to 0055

### DISCUSSION THREAD

**Discussion by iamarifridzuan** [hackintosh-club/intel-nuc10#66] [General]
Hi everyone! I'd like to ask if anybody can share BIOS 0055 for NUC10i7FNH? Currently running on BIOS 0056. Thank you!

**Comment by ghost** [comment]
https://github.com/mackonsti/nuc10i7fnh/tree/master/BIOS

**Reply by iamarifridzuan** [reply]
Thank you very much!
